<!DOCTYPE HTML>
<!--
	Spectral by HTML5 UP
	html5up.net | @ajlkn
	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
-->
<html>
	<head>
		<title>Alamya Health Solutions - First in Class</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
		<link rel="stylesheet" href="assets/css/main.css" />
		<noscript><link rel="stylesheet" href="assets/css/noscript.css" /></noscript>
		<script src="https://kit.fontawesome.com/4dc2e322e8.js" crossorigin="anonymous"></script>
	</head>
	<body class="is-preload">

		<!-- Page Wrapper -->
			<div id="page-wrapper">

				<!-- Header + Menu -->
				<header id="header" class="alt">
					<h1><a href="index.html"><img src="images/ala-logo.png" class="logo"/></a></h1>
					<nav id="nav">
						<ul>
							<li class="special">
								<a href="#menu" class="menuToggle"><span>Menu</span></a>
								<div id="menu">
									<ul>
										<li><a href="index.html">Home</a></li>
										<li><a href="our-story.html">Our Story</a></li>
										<li><a href="technology.html">Technology</a></li>
										<li><a href="about-us.html">Team</a></li>
										<li><a href="partners.html">Partners</a></li>
										<li><a href="resources.html">Resources</a></li>
										<!--li><a href="projects.html">Projects</a></li-->
										<!--li><a href="news-announcements.html">Announcements</a></li-->
										<li><a href="contact-page.html">Contact Us</a></li>
									</ul>
								</div>
							</li>
						</ul>
					</nav>
				</header>


				<!-- Main -->
				<section id="banner">
					<div class="inner">
					<header>
						<h2>First in Class</h2>
					</header>
					<a href="#one" class="more scrolly">Learn More</a>
					</div>
				</section>

				<section id="one" class="wrapper style5">
					<div class="inner">
						<h3>Comprehensive short-read sequencing (SRS) with variant clarification through reflex testing</h3>
						<ul class="product-list">
							<li class="product-item">
								<span class="icon solid fa-dna item-counter"></span>
								<b>Short-read (SR) genome</b> - Includes short-read mitochondrial genome
							</li>
						</ul>
						<h4>Reflex to:</h4>
						<ol class="product-list">
							<li class="product-item">
								<span class="item-counter">1</span>
								<b>Short-read (SR) transcriptome</b>
							</li>
							<li class="product-item">
								<span class="item-counter">2</span>
								<b>Infinium MethylationEPIC microarray</b>
							</li>
						</ol>
						
						<p>
							These tests are performed in a CLIA-certified laboratory with clinical reports reviewed and signed by an ABMGG-certified laboratory geneticist.
						</p>
						
						<h3>Who is this test for?</h3>
						<ul>
							<li>Patients requiring quick test results to guide management</li>
							<li>Patients with complex phenotypes</li>
							<li>Patients with uninformative or inconclusive prior testing</li>
						</ul>
					</div>
				</section>

			<article style="position: relative">
				<div class="product-menu">
					<ul id="product-items" class="product-menu-entries">
						<li><a class="scrolly" href="#seq">Sequencing and Interpretation</a></li>
						<li><a class="scrolly" href="#order">Samples and Ordering</a></li>
						<li><a class="scrolly" href="#report">Clinical Report</a></li>
						<li><a class="scrolly" href="#billing">Billing</a></li>
					</ul>
				</div>
				
				<section id="seq" class="wrapper style2">
					<div class="inner">
						<h3> Sequencing and Interpretation</h3>
						<p>
							The Alamya Health comprehensive short-read sequencing (SRS) genome test provides sequencing of the human genome with reflex to a short-read (SR) transcriptome and/or Infinium MethylationEPIC microarray for interpretation and classification of some variants. Both the SR transcriptome and the Infinium MethylationEPIC microarray may be used to provide support for the interpretive clarification of variants of uncertain clinical significance and/or confirmation of whether a variant is pathogenic or benign.
						</p>
					
						<h4>Sequencing</h4>
						<p>
							Our <b>First in Class</b> genome is sequenced using a NovaSeq 6000 in a CAP/CLIA certified laboratory. The quality control metrics are such that >95% of targeted bases are covered at >15x, >90% of targeted bases are covered at >20x with unique sequencing reads after de-duplication.  The minimum acceptable average read depth is 35x. Bioinformatic analysis and clinical reporting are performed at Breakthrough Genomics using an internally developed pipeline. 
						</p>
						<p>
							All disease-causing variants reported in HGMD® and ClinVar as well as all variants with minor allele frequency (MAF) of less than 1% in gnomAD database are considered. Evaluation is focused on coding exons along with previously reported non-coding and splicing pathogenic or likely pathogenic variants reported in HGMD or ClinVar. All pertinent inheritance patterns are considered. In addition, provided family history and clinical information are used to evaluate identified variants. 
						</p>
						<ul class="interpretation">
							<li id="cnv">
								<h4>Copy Number Variants (CNV), Single Nucleotide Variants (SNV) and Indels</h4>
								<div class="expand-collapse-container">
									<p>
										Three structural variant (SV) calling algorithms are employed (Lumpy, CNVnator, and Manta) that utilize read depth, single nucleotide polymorphism (SNP) information, split reads, and reads which map to two different sites in the genome to detect deletions, duplications, insertions and inversions. The overall sensitivity for SNV and Indels is 99.86% and 99.39%, respectively. Sensitivity for detection of insertions (as opposed to duplications) is currently at approximately 20%. At this time, balanced translocations are not reported.  The ability to detect SVs may be impacted by somatic mosaicism. The detection rate for CNVs ≥2 exons is 95%.
									</p>
									<p>
										Identified SNV/indels and CNVs are evaluated with respect to their pathogenicity and causality and are classified as pathogenic, likely pathogenic, variant of unknown significance (VUS), likely benign, and benign. Variants that are related to the phenotype of the patient and those that are  potentially causative, i.e., classified as pathogenic, likely pathogenic, or VUS, are reported. Variants of relevance identiﬁed by NGS are continuously and individually validated for quality; those variants meeting internal QC criteria (based on extensive validation processes) are not validated by Sanger sequencing. All reported CNVs are confirmed by qPCR.
									</p>
								</div>
							</li>
							<li id="unidisomy">
								<h4>Uniparental Disomy (UPD)</h4>
								<div class="expand-collapse-container">
									<p>
										UPD and parental origin is analyzed based on homozygosity and haplotype analysis.
									</p>
								</div>
							</li>
							<li id="mitochondrial">
								<h4>Mitochondrial Genome</h4>
								<div class="expand-collapse-container">
									<p>
										For mitochondrial genome sequencing, the following quality control metrics are generally achieved: an average read depth of >3,000x and a minimum acceptable read depth of 500x. Variant calls for the mitochondrial genome are made using the Mutserve pipeline and annotated using Alamut batch.  At this time, structural variants within the mitochondrial genome are not reported.
									</p>
								</div>
							</li>
							<li id="transcriptome">
								<h4>Transcriptome</h4>
								<div class="expand-collapse-container">
									<p>
										RNA-seq, also known as transcriptome sequencing, is a valuable addition to the genetic testing. It has been reported to improve the diagnostic rate by 7.5%–36% depending on the clinical phenotype and tissue sample. Furthermore, RNA-seq aids in prioritizing and resolving VUSs (PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/33001864/" target="_blank">33001864</a>).
									</p>
									<p>
										Multiple studies have shown the utility of functional studies such as RNA seq to enable detection of disease-causing non coding or synonymous splicing variants (PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/35946377/" target="_blank">35946377</a>; <a href="https://pubmed.ncbi.nlm.nih.gov/35063693/" target="_blank">35063693</a>). For example, Wai et al (2020; PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/32123317/">32123317</a>) found that 33% of VUSs affect splicing.  Globin depleted and polyA selected transcriptome sequencing from the blood sample is carried out as a reflex and confirmatory test.
									</p>
									<p>
										For results including variants that are potentially causative of altered splicing and of the clinical presentation, transcriptome analysis will be performed to assess the potential impact of the variants. The results of this testing can inform variant classification.
									</p>
								
								</div>
							</li>
							<li id="microarray">
								<h4>Infinium MethylationEPIC microarray</h4>
								<div class="expand-collapse-container">
									<p>
										Epigenetic disruptions play a key role in many congenital disorders, particularly neurodevelopmental disorders. For variants in genes that encode epigenetic regulators and that could be associated with the clinical findings, genomic DNA methylation will be assessed to guide interpretation of pathogenicity. This approach focuses on genes with well-established epigenetic signatures and for which a unique signature can indicate the presence of pathogenic variants. This supplementary testing provides valuable additional information to aid in the classification of variants.
									</p>
									<p>
										We use the Illumina® Infinium Human Methylation EPIC BeadChip platform which contains more than 800 thousand CpG sites for detection of DNA methylation at single CpG resolution. The banked genomic DNA is bisulfite modified and used for microarray interrogation of CpG methylation. Analysis of the distribution of methylated CpGs is based on comparison of the genome-wide DNA methylation profile for the test sample to the reference pathogenic variant and control profiles. Reference pathogenic variant profiles were derived from samples with loss of function variants. Principal component analysis is used to visualize the profile of the test sample in comparison to the distinct profiles of controls and individuals with pathogenic variants in the disease gene.
									</p>
									<p>
										If we find a variant for which assessing DNA methylation would be useful to interpret pathogenicity, the above test will be run to inform variant classification. 
									</p>
								</div>
							</li>
						</ul>
						<h4>Test Limitations</h4>
						<p>
							Test results are interpreted in the context of clinical findings, family history, and other laboratory data.
						</p>
						<ul>
							<li>
								Due to limitations in technology, certain genomic regions either might not be covered or might be poorly covered such that variants cannot be confidently detected.
							</li>
							<li>
								Although next generation sequencing technologies and our bioinformatics analysis significantly reduce the contribution of pseudogene sequences or other highly-homologous sequences, these still occasionally interfere with the technical ability of the assay to identify pathogenic variant alleles in both sequencing and deletion/duplication analyses.
							</li>
							<li>
								Rare polymorphisms might lead to false negative or false positive results.
							</li>
							<li>
								Misinterpretation of results might occur if the information provided is inaccurate or incomplete. If results reported do not match the clinical findings, additional testing should be considered.
							</li>
							<li>
								Specific genetic events like CNVs, translocations and repeat expansions might not be reliably detected with Whole Genome SRS.
							</li>
						</ul>
						<p>
							Transcriptome sequencing and the Infinium MethylationEPIC microarray are currently used to generate clarifying or supporting evidence and are not accredited or licensed as independent diagnostic assays. Classification of gene variants using data generated by either or both of these tests is based on our current understanding of RNA splicing/expression and distinct DNA methylation profiles that can be associated with variants in the specific gene. Such variants may be reclassified over time based on reliable new information.
						</p>
					</div>
				</section>
				
				<section id="order" class="wrapper style5">
					<div class="inner">
						<h3>Samples and Ordering</h3>
						<h4>Sample Requirements</h4>

						<div class="table-container">
							<table class="sample-instructions">
								<colgroup>
									<col span="1" style="width: 20%;">
									<col span="1" style="width: 80%;">
								</colgroup>
								<thead>
								</thead>
								<tbody>
									<tr>
										<th>Whole Genome</th>
										<td>Whole blood* (1 EDTA tube); 3.0-5.0mL (0.6mL min)</br>
											<i>OR</i></br>
											Saliva (Sponge kits and spit kits available)</br>
											<i>OR</i></br>
											DNA Isolated from whole blood: 1µg at 100ng/µl in TE, A260/280 = 1.7-2.0µg
										</td>
									</tr>
									<tr>
										<th>Transcriptome</th>
										<td>Whole blood (1 PAXgene/Tempus RNA tube): 4.0mL (2.0mL min)</td>
									</tr>
									<tr>
										<th>Infinium MethylationEPIC microarray**</th>
										<td>Whole blood (1 EDTA tube): 3.0-5.0mL (1.0mL min)</br>
											<i>OR</i></br>
											DNA isolated from whole blood: min of 1µg</td>
									</tr>
								</tbody>
								<tfoot>
									<tr>
										<td ><sup>*</sup>Preferred</td>
										<td ><sup>**</sup>Residual post-test DNA (isolated for whole genome sequencing) can be used for the Infinium MethylationEPIC microarray depending on the quantity and quality.
										</td>
									</tr>
								</tfoot>
							</table>
							</div>

						<p>Transcriptome sequencing and/or the Infinium MethylationEPIC microarray might not be indicated for result interpretation; however, urgent test analyses are facilitated by the availability of all samples. If your patient requires a rapid result, please send:</p>
						<ul>
							<li><b>1 EDTA tube of whole blood for the genome</b></li>
							<li><b>1 RNA tube of whole blood for the transcriptome</b></li>
						</ul>

						<h4>Shipping and Storage</h4>
						<ul>
							<li>Samples should be shipped at ambient temperature (18-25°C) or refrigerated (2-8°C).</li>
							<li>Whole blood is stable for 5 days at ambient temperature (18-25°C) or 7 days refrigerated (2-8°C). <b>Do not freeze.</b></li>
							<li>Saliva is stable for 2 years at 2-8°C. <b>Do not freeze.</b></li>
							<li>Extracted DNA is stable for 5 years at 2-8°C. DNA can also be stored frozen at -25°C to -15°C.</li>
						</ul>

						<ul class="features"><!-- To do: rename class -->
							<li class="icon solid fa-clipboard-list">
								<h4>Order a test</h4>
								<p>
									As with any genetic testing, it is important to provide a detailed patient phenotype and clinical history, including previous testing (positive or negative) and relevant family history. Our requisition provides a phenotype checklist, space for a family history and previous testing.
								</p>
								<div>
									<a href="./order-sr.html" class="button primary">Order</a>
								</div>
							</li>
							<li class="icon solid fa-boxes-packing">
								<h4>Ship samples to:</h4>
								<address>
									Alamya Health<br/>
									℅ Breakthrough Genomics<br/>
									2 Hughes, #100<br/>
									Irvine, CA 92618<br/>
								</address>
							</li>
						</ul>
						
					</div>
				</section>
				
				<section id="report" class="wrapper style2">
					<div class="inner">
						<h3>Clinical Report</h3>
						<p>
							Comprehensive reporting of the genomic data is of the utmost importance to provide insights into diagnostic certainty, additional confirmatory testing, disease natural history, and available therapeutics.
							All variants in genes potentially related to the proband's medical condition are reported.
						</p>
						<p>
							In line with ACMG recommendations for reporting of secondary findings in clinical exome and genome sequencing (PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/34012069/" target="_blank">34012069</a>), we report secondary findings, i.e., pathogenic variants and likely pathogenic variants, only in the recommended genes and for the recommended phenotypes.
						</p>
						<a href="documents/ALA_Sample-Report.pdf" target="_blank" class="button primary">View Sample Clinical Report</a>
					</div>
				</section>

				<section id="billing" class="wrapper style5">
					<div class="inner">
						<h3>Billing</h3>
						<h4>Test pricing and CPT codes</h4>

						<div class="table-container">
						<table class="pricing">
							<thead>
								<tr>
									<th>Test name</th>
									<th>Charge</th>
									<th>CPT</th>
								</tr>
							</thead>
							<tbody>
								<tr>
									<th>Short-Read genome, proband analysis only</td>
									<td>US$ <b>2,100</b></td>
									<td>81425</td>
								</tr>
								<tr class="reflex">
									<td>Reflex where applicable: Short-Read transcriptome</td>
									<td>US$ <b>0</b></td>
									<td>N/A</td>
								</tr>
								<tr class="reflex">
									<td>Reflex where applicable: EPIC array</td>
									<td>US$ <b>0</b></td>
									<td>N/A</td>
								</tr>

								<tr>
									<th>Short-Read genome, trio analysis</td>
									<td>US$ <b>5,000</b></td>
									<td>81425 x 1<br/>81426 x 2</td>
								</tr>
								<tr class="reflex">
									<td>Reflex where applicable: Short-Read transcriptome</td>
									<td>US$ <b>0</b></td>
									<td>N/A</td>
								</tr>
								<tr class="reflex">
									<td>Reflex where applicable: Infinium MethylationEPIC array</td>
									<td>US$ <b>0</b></td>
									<td>N/A</td>
								</tr>

								<tr>
									<th>Rapid Short-Read genome, proband analysis only</td>
									<td>US$ <b>2,700</b></td>
									<td>81425</td>
								</tr>
								<tr class="reflex">
									<td>Reflex where applicable: Short-Read transcriptome</td>
									<td>US$ <b>0</b></td>
									<td>N/A</td>
								</tr>
								<tr class="reflex">
									<td>Reflex where applicable: EPIC array</td>
									<td>US$ <b>0</b></td>
									<td>N/A</td>
								</tr>

								<tr>
									<th>Rapid Short-Read genome, trio analysis</td>
									<td>US$ <b>6,100</b></td>
									<td>81425 x 1<br/>81426 x 2</td>
								</tr>
								<tr class="reflex">
									<td>Reflex where applicable: Short-Read transcriptome</td>
									<td>US$ <b>0</b></td>
									<td>N/A</td>
								</tr>
								<tr class="reflex">
									<td>Reflex where applicable: Infinium MethylationEPIC array</td>
									<td>US$ <b>0</b></td>
									<td>N/A</td>
								</tr>
							</tbody>
						</table>
						</div>

						<ul>
							<li>
								When a variant of significance is found in either the proband or the trio, SR transcriptome/Infinium MethylationEPIC microarray may be performed at no charge for all orders received
							</li>
							<li>
								For all in-patient testing, Alamya Health’s partner laboratory must bill the institution directly
							</li>
							<li>
								For outpatient/clinic testing, Alamya Health’s partner laboratory will bill either the provider/institution or the patient’s insurance per the selection indicated on the requisition
							</li>
						</ul>
					</div>
				</section>

				<a id="go-top" href="#one" class="scrolly" title="Back to the test overview" ></a>
			</article>

				<!-- Footer -->
					<footer id="footer">
						<!-- ul class="icons">
							<li><a href="#" class="icon brands fa-twitter"><span class="label">Twitter</span></a></li>
							<li><a href="#" class="icon brands fa-facebook-f"><span class="label">Facebook</span></a></li>
							<li><a href="#" class="icon brands fa-instagram"><span class="label">Instagram</span></a></li>
							<li><a href="#" class="icon brands fa-dribbble"><span class="label">Dribbble</span></a></li>
							<li><a href="#" class="icon solid fa-envelope"><span class="label">Email</span></a></li>
						</ul-->
						<ul class="copyright">
							<li>&copy; Alamya Health 2023</li><li>Design: <a href="http://html5up.net">HTML5 UP</a></li>
						</ul>
					</footer>

			</div>

		<!-- Scripts -->
			<script src="assets/js/jquery.min.js"></script>
			<script src="assets/js/jquery.scrollex.min.js"></script>
			<script src="assets/js/jquery.scrolly.min.js"></script>
			<script src="assets/js/browser.min.js"></script>
			<script src="assets/js/breakpoints.min.js"></script>
			<script src="assets/js/util.js"></script>
			<script src="assets/js/main.js"></script>
			<script src="assests/js/expandCollapse.js"></script>

	</body>
</html>