<!DOCTYPE HTML>
<!--
	Spectral by HTML5 UP
	html5up.net | @ajlkn
	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
-->
<html>
	<head>
		<title>Alamya Health Solutions - Generation Beyond</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
		<link rel="stylesheet" href="assets/css/main.css" />
		<noscript><link rel="stylesheet" href="assets/css/noscript.css" /></noscript>
		<script src="https://kit.fontawesome.com/4dc2e322e8.js" crossorigin="anonymous"></script>
	</head>
	<body class="is-preload">

		<!-- Page Wrapper -->
			<div id="page-wrapper">


				<!-- Header + Menu -->
				<header id="header" class="alt">
					<h1><a href="index.html">alamya health</a></h1>
					<nav id="nav">
						<ul>
							<li class="special">
								<a href="#menu" class="menuToggle"><span>Menu</span></a>
								<div id="menu">
									<ul>
										<li><a href="index.html">Home</a></li>
										<li><a href="our-story.html">Our Story</a></li>
										<li><a href="solutions.html">Solutions</a></li>
										<li><a href="technology.html">Technology</a></li>
										<li><a href="about-us.html">Team</a></li>
										<li><a href="partners.html">Partners</a></li>
										<li><a href="resources.html">Resources</a></li>
										<!--li><a href="projects.html">Projects</a></li-->
										<!--li><a href="news-announcements.html">Announcements</a></li-->
										<li><a href="contact-page.html">Contact Us</a></li>
									</ul>
								</div>
							</li>
						</ul>
					</nav>
				</header>


				<!-- Main -->
				<section id="banner">
					<div class="inner">
					<header>
					    <p><a href="solutions.html">Alamya Solutions</a></p>
						<br>
						<h2>Generation Beyond</h2>
						<!--p>
							Clinical short-read genome sequencing with a turnaround time of 4-6 weeks
						<br><br>
							These tests are performed in a CLIA-certified lab with clinical reports reviewed and signed by an ABMGG-certified laboratory geneticist
						</p-->
					</header>
					<a href="#one" class="more scrolly">Learn more.</a>
					</div>
				</section>

				<section id="one" class="wrapper style5">
					<div class="inner">
						<h3>Research-grade Nanopore long-read whole genome sequencing</h3>
						<p>Alamaya Health’s <b>research-grade</b> Nanopore long-read (LR) whole genome sequencing, which includes DNA methylation profiling, and Nanopore LR transcriptome sequencing (cDNA) is the next generation of genome profiling. A research report is available as are the raw data files (with some limitations). Turnaround times for test results are rapid relative to comparable test offerings and are dependent on reports and data required.
						</p>
						
						
						<h3>Who is this test for?</h3>
						<ul>
							<li>Clinical research participants requiring LR human genome sequencing +/- LR transcriptome</li>
						</ul>
					</div>
				</section>

			<article style="position: relative">
				<div class="product-menu">
					<ul id="product-items" class="product-menu-entries">
						<!--li><a class="scrolly" href="#one">First in Class</a></li-->
						<li><a class="scrolly" href="#seq">Sequencing and Interpretation</a></li>
						<li><a class="scrolly" href="#order">Ordering and Costs</a></li>
					</ul>
				</div>
				
				<section id="seq" class="wrapper style2">
					<div class="inner">
						<h3> Sequencing and Interpretation</h3>
	 
						<p>
							The Alamya Health comprehensive LRS (long-read sequencing) genome research test provides sequencing of the entire human genome, as well as a LR transcriptome.
						</p>
					
						<h4>Genome Sequencing</h4>
						<p>
							Our Generation Beyond genome sequencing test is performed on the Oxford Nanopore Technologies PromethION platform. This sequencing assay can identify single nucleotide variants (SNVs) and small insertions and deletions (indels) in regions of the genome inaccessible by short-read technologies. It also profiles short tandem repeat expansions, structural variants (SVs) such as insertions, inversions, and translocations, and copy number variants (CNVs).
						</p>
						<p>
							Basecalling, mapping, variant calling, and phasing are performed using the EPI2ME pipeline developed by Oxford Nanopore Technologies. This pipeline leverages the latest production-level bioinformatics tools to generate a comprehensive set of genomic variants. Details of the analysis pipeline can be found at <a target="_blank" href="https://github.com/epi2me-labs/wf-human-variation"> https://github.com/epi2me-labs/wf-human-variation</a>. Briefly, canonical and modified basecalling is performed with Dorado and Remora; alignment is performed with minimap2 against the GRCh38 reference genome; methylated bases are extracted with modbam2bed; SNVs and indels are called with Clair3; SVs are called with Sniffles2; repeat expansions are called by Straglr; and CNVs are called with QDNAseq.
						</p>
						<p>
							SNVs, indels, and CNVs are annotated and prioritized using the ENLITER toolkit from Breakthrough Genomics. ENLITER integrates variant interpretation and annotation with current literature, known genotype-to-phenotype databases (e.g. ClinVar, OMIM, and DECIPHER), population allele frequencies from gnomAD, and <i>in silico</i> scoring metrics (e.g. SpliceAI and CADD).
						</p>
						<p>
							SVs, repeat expansions, and aberrant allelic methylation are annotated against known disease genes and disease-associated imprinted regions and compared to public allele frequency databases (e.g., gnomAD and 1000 Genomes project) as well as an in-house database.
						</p>
						<p>
							Known or predicted pathogenic variants and rare variants predicted to alter gene expression, splicing, or function are prioritized for review. In situations where the candidate gene has detectable expression in blood, transcriptome sequencing data will be used to support variant classification.
						</p>

						<h4>Transcriptome Sequencing</h4>
						<p>
							The Oxford Nanopore Technologies PromethION platform assay involves cDNA sequencing from whole blood RNA depleted for hemoglobin mRNA and ribosomal RNAs. Nanopore LRS enables characterization of complete transcripts and facilitates classification of candidate variants with molecular evidence of aberrant splicing, expression, and allele-specific expression.
						</p>
						<p>
							Canonical and modified base calling is performed with Dorado. Mapping, transcript assembly, and gene-level and transcript-level quantification will be performed using the EPI2ME pipeline developed by Oxford Nanopore Technologies. This pipeline leverages the latest production-level bioinformatics tools to generate a comprehensive view of an individual’s transcriptome. Details of the analysis pipeline can be found at <a target="_blank" href="https://github.com/epi2me-labs/wf-transcriptomes">https://github.com/epi2me-labs/wf-transcriptomes</a>. Briefly, FASTQ files are preprocessed with PyChopper, transcript-guided alignment is performed with minimap2 against the GRCh38 reference genome with the Gencode (v43) annotation database, transcript assembly is performed via stringtie, compared against the reference database using gffcompare, and fusion genes are detected by JAFFA.
						</p>
						<p>
							Candidate gene expression, detected splicing isoforms, and biased allelic expression is investigated on a gene-by-gene basis guided by variants identified through the Alamya Generation Beyond genome or to help resolve variants of uncertain significance in patients who have received clinical sequencing. This test might be unable to determine the consequence of candidate variants in genes with a low level of expression in blood.
						</p>
						
						<h4>Limitations</h4>
						<p>
							<b>This assay is performed for the purpose of research and does not constitute a clinical test.</b>
						</p>
						<ul>
							<li>
								Absence of a pathogenic variant does not exclude a genetic basis for disease.
							</li>
							<li>
								Sensitivity for detecting variants varies among genomic regions and variant types such that the causal variant may not be detected.
							</li>
							<li>
								Interpretation of genetic variation is based on current biological and clinical knowledge of gene-disease associations and current understanding of the association with phenotype and family history.
							</li>
							<li>
								Re-interpretation of the data can be requested in light of additional patient information or at least six months after the report date.
							</li>
						</ul>
					</div>
				</section>
				
				<section id="order" class="wrapper style5">
					<div class="inner">
						<h3>Ordering and Costs</h3>

						<p>
							Please contact us if you have questions around whether this test is right for your patient or project at <script>user = 'sales';site = 'alamyahealth.com';document.write('<a href=\"mailto:' + user + '@' + site + '\">');document.write(user + '@' + site + '</a>');</script> or call <script>number = '+1 (628) 252-5243';document.write('<a href=\"tel:' + number + '\">');document.write(number + '</a>');</script>.
						</p>
					</div>
				</section>

				<a id="go-top" href="#one" class="scrolly" title="Back to the test overview" ></a>
			</article>

				<!-- Footer -->
					<footer id="footer">
						<!-- ul class="icons">
							<li><a href="#" class="icon brands fa-twitter"><span class="label">Twitter</span></a></li>
							<li><a href="#" class="icon brands fa-facebook-f"><span class="label">Facebook</span></a></li>
							<li><a href="#" class="icon brands fa-instagram"><span class="label">Instagram</span></a></li>
							<li><a href="#" class="icon brands fa-dribbble"><span class="label">Dribbble</span></a></li>
							<li><a href="#" class="icon solid fa-envelope"><span class="label">Email</span></a></li>
						</ul-->
						<ul class="copyright">
							<li>&copy; Alamya Health 2023</li><li>Design: <a href="http://html5up.net">HTML5 UP</a></li>
						</ul>
					</footer>

			</div>

		<!-- Scripts -->
			<script src="assets/js/jquery.min.js"></script>
			<script src="assets/js/jquery.scrollex.min.js"></script>
			<script src="assets/js/jquery.scrolly.min.js"></script>
			<script src="assets/js/browser.min.js"></script>
			<script src="assets/js/breakpoints.min.js"></script>
			<script src="assets/js/util.js"></script>
			<script src="assets/js/main.js"></script>
			<script src="assests/js/expandCollapse.js"></script>

	</body>
</html>